app/ui-clean-history/client/views/cleanHistory.js

```diff
@@ -55,6 +56,10 @@ Template.cleanHistory.helpers({
 		const room = ChatRoom.findOne(Session.get('openedRoom'));
 		return room && room._id;
 	},
+	roomType() {
```

Comment:
Just one last thing: where is this being used? I couldn't find it. If it isn't used anywhere, please remove

Reply:
It's being used here - ```suggestion
	const purgeWorker = function(roomId, roomType, oldest, latest, inclusive, limit, excludePinned, ignoreDiscussion, filesOnly, fromUsers)
```

Reply:
I see you used a different one here - `result = await purgeWorker(room._id, room.t, from...`

Reply:
Well, you made it a helper. Helpers are used inside the HTML usually.
If you pay some extra attention, you'll see that you added `roomType` as a parameter, and didn't actually call for your method. (You can't call functions in function definitions). I made a couple commits on your branch, and already fixed that (take a look [here](https://github.com/RocketChat/Rocket.Chat/pull/16728/commits/bfe14242e17936ee2cd3b9cd9913ac8db025d817))

Thanks for your help!

Reply:
Apologies for being a bit rough with this PR. I removed the item as requested and tested functionality. Looking forward to seeing this merged!